.github/workflows/ci.yml

```diff
@@ -56,34 +54,28 @@ jobs:
         with:
           prefix-key: "0" # change this to invalidate CI cache
           shared-key: "cargo_nextest-${{ matrix.cache_id }}"
-      - run: cargo llvm-cov show-env | tr -d "'" >> "$GITHUB_ENV"
 
       # Run unit tests
-      - run: cargo nextest run --locked --workspace --exclude integration-tests --cargo-profile dev-release --profile ci ${{ matrix.flags }}
+      - run: just codecov "nextest-unit ${{ matrix.type }}"
         env:
           RUST_BACKTRACE: short
-      - run: cargo llvm-cov report --profile dev-release --codecov --output-path unittests.json
       - uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          files: unittests.json
+          files: codecov.json
           fail_ci_if_error: true
           flags: unittests,${{ matrix.id }}
       # See https://github.com/taiki-e/cargo-llvm-cov/issues/292
       - run: find target -name '*.profraw' -delete
```

Comment:
Should this not be a part of the Justfile as well? And then perhaps check if `--no-clean` is passed to `just` and disable the this delete there?

Reply:
Hmm so it’s using the codecov token, which allows uploading coverage reports for any PR. So it’d be bad to make it public. From that on, I don’t see the point in making this part of the Justfile.

Reply:
Oooooh yes ok I get it, I thought it was for the `codecov-action` 😅 This is now implemented!